drivers/note/note_driver.c

```diff
@@ -435,6 +455,122 @@ static inline int note_isenabled_dump(void)
 }
 #endif
 
+#if CONFIG_TASK_NAME_SIZE > 0
+
+/****************************************************************************
+ * Name: note_read_taskname
+ *
+ * Description:
+ *   read task name info from the specified position in buffer
+ *
+ * Input Parameters:
+ *   pos - position
+ *   info - task name info buffer
+ *
+ * Returned Value:
+ *   None
+ *
+ ****************************************************************************/
+
+static void note_read_taskname(size_t pos,
+                               FAR struct note_taskname_info_s *info)
+{
+  size_t space;
+  pos = pos % CONFIG_DRIVER_NOTERAM_TASKNAME_BUFSIZE;
+  space = CONFIG_DRIVER_NOTERAM_TASKNAME_BUFSIZE - pos;
+  if (space > sizeof(struct note_taskname_info_s))
+    {
+      space = sizeof(struct note_taskname_info_s);
+    }
+
+  memcpy(info, g_note_taskname.buffer + pos, space);
+  memcpy(((void *)info) + space, g_note_taskname.buffer,
```

Comment:
```suggestion
  memcpy(((char *)info) + space, g_note_taskname.buffer,
```
since if I recall correctly `(void *)` increment is undefined

Reply:
done